milli/src/search/facet/filter.rs

```diff
@@ -266,11 +264,12 @@ impl<'a> Filter<'a> {
     fn evaluate_operator(
         rtxn: &heed::RoTxn,
         index: &Index,
-        numbers_db: heed::Database<FacetLevelValueF64Codec, CboRoaringBitmapCodec>,
-        strings_db: heed::Database<FacetStringLevelZeroCodec, FacetStringLevelZeroValueCodec>,
         field_id: FieldId,
         operator: &Condition<'a>,
     ) -> Result<RoaringBitmap> {
+        let numbers_db = index.facet_id_f64_docids;
+        let strings_db = index.facet_id_string_docids;
+
```

Comment:
This change is only here to make the code clearer, right?

Reply:
Yep, I noticed we were passing this by parameters everywhere when it was actually available directly from the `Index` without any function call (which has _similar_ performance).
And it’s easier to read without theses two parameters 😁